packages/neon-core/src/sc/manifest/ContractPermission.ts

```diff
@@ -0,0 +1,49 @@
+export interface ContractPermissionLike {
+  contract: string;
+  methods: "*" | string[];
+}
+
+export class ContractPermission {
+  public contract: string;
+  public methods: "*" | string[];
+
+  public constructor(obj: Partial<ContractPermissionLike> = {}) {
+    const { contract = "*", methods = "*" } = obj;
+    this.contract = this._formatContract(contract);
+    this.methods = methods;
+  }
+
+  public get isHash(): boolean {
```

Comment:
A getter will be called like a property: `permission.isHash`. Is this the intention? In JS the convention would be methods for this: `permission.isHash()`.

Reply:
This is the intention.